src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -56,6 +56,24 @@ public RangeConstraint(IComparable from, IComparable to) : base( from, to )
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="RangeConstraint"/> class for objects that did not implement <see cref="IComparable"/>.
+        /// </summary>
+        /// <remarks>from must be less than or equal to true</remarks> 
+        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
```

Comment:
[Following up](https://github.com/nunit/nunit/pull/1855#discussion_r115353617)
What about `Must be less than or equal to the <paramref name="to"/> value.`?